external/java/src/test/java/ch/epfl/dedis/byzcoin/ByzCoinRPCTest.java

```diff
@@ -119,6 +123,22 @@ void subscribeSkipBlocks() throws Exception {
         bc.unsubscribeBlock(receiver);
     }
 
+    /**
+     * Subscribe to new blocks using a stream
+     */
+    @Test
+    void subscribeSkipBlockStream() throws Exception {
+        Stream<SkipBlock> stream = bc.subscribeSkipBlock();
+
+        // create one block
+        bc.getGenesisDarcInstance().evolveDarcAndWait(bc.getGenesisDarc(), admin, 10);
```

Comment:
what's more interesting to do is use 0 instead of 10 in `evolveDarcAndWait`. it'll return immediately but the stream should block until a new block is accepted.